doc-content/enterprise-only/installation/installer-run-proc.adoc

```diff
@@ -123,7 +123,7 @@ Make a note of the user name and password. You will need them to access {CENTRAL
 . When the installation has completed, click *Next*.
 
 ifdef::PAM[]
-. On the *Configure Runtime Environment* page, choose whether to perform the default installation or perform an advanced configuration.
+. If you installed {KIE_SERVER}, the *Configure Runtime* step appears under *Component Installation*. On the *Configure Runtime Environment* page, choose whether to perform the default installation or perform an advanced configuration.
```

Comment:
I would suggest ... "If {KIE_SERVER} is installed, the *Configure Runtime*...". Also, we can remove whether from the later part of the sentence like "....choose to perform the default installation or perform an advanced configuration."